LottoDairy/CalendarScene/Views/HeaderView/YearAndMonthView.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  YearAndMonthView.swift
+//  LottoDairy
+//
+//  Created by Brody on 2023/08/18.
+//
+
+import UIKit
+
+final class YearAndMonthView: UIStackView {
+    
+    var yearLabel: LottoLabel = {
+        let label = LottoLabel(text: "2023", font: .gmarketSans(size: .title2, weight: .light))
```

Comment:
```suggestion
        let label = LottoLabel(font: .gmarketSans(size: .title2, weight: .light))
```

```swift
final class LottoLabel: UILabel {
    
    init(text: String? = nil, font: UIFont, textColor: UIColor = .white) {
        super.init(frame: .zero)
        self.text = text
        self.font = font
        self.textColor = textColor
        self.adjustsFontForContentSizeCategory = true
    }
    
    required init?(coder: NSCoder) {
        fatalError("init(coder:) has not been implemented")
    }
}
```

`LottoLabel`의 init에 nil을 default value로 설정하고, 해당 라벨의 text 값을 추후 주는 것으로 구현해보는 건 어떨까요?!